instance/github.py

```diff
@@ -86,6 +87,18 @@ def get_settings_from_pr_body(pr_body):
         return ''
 
 
+def get_ephemeral_databases_from_pr_body(pr_body):
```

Comment:
The name of this method makes it look like it returns databases objects - maybe something like `is_pr_body_requesting_ephemeral_databases()`?